submissions/zhenyakornilov/memory-pair-game/js/index.js

```diff
@@ -0,0 +1,107 @@
+import cards from "./cards.js";
+
+const cardsMenu = document.getElementById("game-cards");
+const totalTriesCounter = document.getElementById("total-flips");
+const states = {
+  cardsMatches: 0,
+  firstFlippedCard: null,
+  secondFlippedCard: null,
+  currentCardPair: [],
+  totalTries: 0,
+};
+const allGameCards = shuffleCards([...cards, ...cards]);
+renderCards(allGameCards, cardsMenu);
+
+function shuffleCards(cardsArr) {
+  for (let i = cardsArr.length - 1; i > 0; i--) {
```

Comment:
Looks `sort` array method is the best choice here

Reply:
Did you mean that Fisher–Yates shuffle is overkill to shuffle game cards? :)

Reply:
We have a built-in `sort` method in JS for this purpose, no need to reinvent the wheel, also it is much better for code readability.

Reply:
But you can leave it, I changed my mind :))